app/views/wizards/claims/add_mentor_wizard/_no_results_step.html.erb

```diff
@@ -0,0 +1,13 @@
+<% caption = current_user.support_user? ? "#{t(".caption")} #{contextual_text}" : t(".caption") %>
+<% content_for :page_title, t(".page_title", trn: wizard.steps[:mentor].trn, contextual_text: caption) %>
+<div class="govuk-width-container">
+  <span class="govuk-caption-l"><%= caption %></span>
```

Comment:
Same here with needing to show the school name in the captions and page title, if the user is a support user.